src/mappings/farms.ts

```diff
@@ -70,10 +72,14 @@ export async function farmStored(
     newIP.farm = newFarm
 
     newFarm.publicIPs?.push(newIP)
-
+    newFarm.totalIps += 1
```

Comment:
This could be more error-prone, and susceptible to overwrite problems (not atomic).
As explained before, i would suggest relay on db `count`  function whenever we need such info, or consider using an indexed view for IPs counters.